.gitignore

```diff
@@ -23,6 +23,7 @@ settings.json
 settings-dev.json
 settings-dev-livedb.json
 settings-dev-live.json
+settings*
```

Comment:
If you're including settings*, can we remove the other settings files?

Reply:
Yeah, that should be the case. Will do it.